src/components/editor/editor-window/key-map.ts

```diff
@@ -0,0 +1,93 @@
+import CodeMirror, { Editor, KeyMap, Pass } from 'codemirror'
+import {
+  makeSelectionBold,
+  makeSelectionItalic,
+  markSelection,
+  strikeThroughSelection,
+  underlineSelection
+} from './tool-bar/utils'
+
+const isMac = navigator.platform.includes('Mac')
```

Comment:
```suggestion
const isMac = navigator.platform.toLowerCase().includes('mac')
```
I think this wouldn't hurt. We never know if every browser will use "Mac".

Reply:
Well according to the Living HTML Standard the value should currently always be 'MacIntel'